app/helpers/intl-get.js

```diff
@@ -7,6 +7,8 @@ import Ember from 'ember';
 import { Stream, read, readHash, destroyStream } from 'ember-intl/utils/streams';
 
 export default function (value, options) {
+    Ember.assert('intl-get helper must be used as a subexpression', options.isInline === true);
```

Comment:
Future versions I plan to deprecate intl-get with a warning in favor of a `t` helper.  Intl-get no longer shares the same concepts of its sibling libs since it only is used to resolve translations keys.

`(intl-get 'messages.foo.bar')` will become `(t 'foo.bar')`

This change will likely happen when I rename `locales` -> `translations`